bitrise.yml

```diff
@@ -4,10 +4,14 @@ project_type: ios
 pipelines:
   pipeline_build_and_test:
     stages:
-    - pre_configuration: {}
-    - build_applications: {}
-    - run_tests: {}
+    - l10n_screenshots: {}
+    # - pre_configuration: {}
+    # - build_applications: {}
+    # - run_tests: {}
 stages:
+  l10n_screenshots:
+    workflows:
+    - L10nBuild: {}
```

Comment:
Temporary workaround to test the changes in `L10nBuild` on a push to the branch.

Reply:
Once we are happy with the `L10nBuild`'s fix, we can revert the changes between lines 7 and 14.

Reply:
As commented below, there is an easier way to try a workflow by changing the trigger

Reply:
Please undo these changes